scripts/setup-testbed.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -ex
+
+cd $(dirname "$0")
+
+# test against 1.1.0 by default
+VERSION="${1:-1.1.0}"
+
+case $VERSION in
+  0.9)
+    VERSION="0.9.0.0";;
+  0.10)
+    VERSION="0.10.2.1";;
+  0.11)
+    VERSION="0.11.0.2";;
+  1.*)
+    VERSION="1.1.0";;
+  *)
+    true
+esac
+
+case $VERSION in
+  0.9*)
+    DOWNLOAD_URL_PREFIX="https://archive.apache.org/dist/kafka";;
+  *)
+    DOWNLOAD_URL_PREFIX="https://apache.org/dist/kafka/";;
+esac
+
+IMAGE="kafka_protocol_test:${VERSION}"
+sudo docker build --build-arg VERSION=$VERSION --build-arg DOWNLOAD_URL_PREFIX=$DOWNLOAD_URL_PREFIX -t $IMAGE .
```

Comment:
Avoid sudo in scripts. Either run the whole script as root or check that the user is in docker group